frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java

```diff
@@ -62,7 +61,10 @@ public static List<String> getBranchNamesForPsiMacheteFile(PsiFile psiFile) {
   @UIEffect
   public static void saveDocument(PsiFile file) {
     val fileDocManager = FileDocumentManager.getInstance();
-    fileDocManager.saveDocument(Objects.requireNonNull(fileDocManager.getDocument(file.getVirtualFile())));
```

Comment:
Unrel: this might end up in NPE in runtime, not much better than having no check at all